### DISCUSSION TITLE
Inverter Power sensor some time gives wrong data ( 65 000 W )

### DISCUSSION THREAD

**Discussion by LionBS76** [vladyspavlov/esphome-must-inverter#6] [General]
Hello,
I have solar inverter MUST PH18-3224.
When I use this project (yaml config for ESHome) all works perfect.
When invertor work from battery without grid, the sensor Inverter Power gives some reasonable data (at least 80 W).
When the grid is available the sensor Invertor Power start giving 65000 W.

![image](https://github.com/user-attachments/assets/3f03eb04-bcb6-4635-9777-7e68baf7ab93)

Is it my personal bug of invertor or some specific of this particular model ?

**Comment by vladyspavlov** [comment]
Change the `value_type` of the sensor to the S_WORD.
https://github.com/vladyspavlov/esphome-must-inverter/blob/main/esp32-pv18.yaml#L312
